ZCS-1248 : add attribute which was there in 877 and deprecate it as it is not used anymore. corrected deprecateDesc for one of the attributes

chinmay15 opened the pull request:
Changes done:
===========
1) added zimbraAdminSieveFeatureVariablesEnabled ldap attribute which was there in 877
2) deprecated the same, as it is not used anymore
3) corrected deprecateDesc for zimbraSieveFeatureVariablesEnabled

Testing done:
==========
1) zimbraAdminSieveFeatureVariablesEnabled comes as deprecated in "zmprov desc -a zimbraAdminSieveFeatureVariablesEnabled"
2) zimbraSieveFeatureVariablesEnabled comes as deprecated with new description in "zmprov desc -a zimbraSieveFeatureVariablesEnabled"

rohan-ambasta commented on store/conf/attrs/zimbra-attrs.xml:
```diff
@@ -9262,14 +9262,20 @@ TODO: delete them permanently from here
   <globalConfigValue>FALSE</globalConfigValue>
   <desc>Whether to enable the Sieve "Variables" extension defined in RFC 5229 in the user-defined sieve rule.</desc>
-  <deprecateDesc>deprecated in favor of zimbraSieveRejectMailEnabled, which can be used at account level</deprecateDesc>
+  <deprecateDesc>deprecated as this attribute is not used anymore and variable feature is availabe to use for all.</deprecateDesc>
```
I think only the following description will be sufficient - 
"Deprecated as this attribute is not used anymore"
Whether the variable feature is available for all or not will be part of documentation right?
OR may be you can rephrase as below  - 
"Variable feature is always enabled, hence this attribute has been deprecated"

chinmay15 replied:
Will update PR with "Variable feature is always enabled, hence this attribute has been deprecated"

chinmay15 commented:
Merged code into develop branch.
commit 62005a66c316c6f3d03defbdbc0ad1d714d022e0
Author: Chinmay Pathak <15.chinmay@gmail.com>
Date:   Thu May 4 11:55:37 2017 +0530

    ZCS-1248 : add attribute which was there in 877 and deprecate it as it is not used anymore. corrected deprecateDesc for one of the attributes

    ZCS-1248 : resolving review comment. updated deprecation description
